### DISCUSSION TITLE
Cheats "Unstable"

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#16110] [Q&A] [answered]
The unstable tag have been on a few cheats for quite a few years, but they all seem to be pretty stable. Is that pop up still necessary, or should I remove them?

**Comment by duncanspumpkin** [comment, marked as answer]
No, there are still many very unstable features of the cheats.

**Reply by karst** [reply]
Okay! Just making sure ☺️
